src/ui/components/Notice/index.js

```diff
@@ -49,10 +49,12 @@ const Notice = ({
     <div className={finalClass}>
       <div className="Notice-icon" />
       <div className="Notice-column">
-        <p className="Notice-text">
-          {children}
-        </p>
-        {actionButton}
+        <div>
```

Comment:
This is the best idea I could come up with but I'm open to alternatives. My intention was to use flex in the container for vertical centering but continue relying on inline block display rules to render the message and action button correctly (i.e. not apply flex to the message and the button).

Reply:
I think it's fine 😄